webapp/src/cardFilter.ts

```diff
@@ -77,43 +77,43 @@ class CardFilter {
             if (filter.values.length === 0) {
                 return true
             }
-            return filter.values[0] === value
+            return filter.values[0].toLowerCase() === value
```

Comment:
We should add the safety operator if we access the values directly:

```suggestion
            return filter.values?[0].toLowerCase() === value
```

Same for all the other accesses

Reply:
Okay